client/src/ui/deckbuilder/DeckBuilderColumn.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div
+      class="column"
+      @dragover="dragOver"
+      @dragleave="dragEnd"
+      @dragend="dragEnd"
+      @drop="drop"
+  >
+    <div
+        v-for="(card, index) in column"
+        class="card"
+    >
+      <img
+          class="card-img"
+          :src="getImageSrc(card)"
+          :alt="card.name"
+          :data-index="index"
+          @dragstart="dragStart"
```

Comment:
Should this listener be on the parent div? Also mozilla dev claims that the element also needs to have `draggable="true"` set on it, but I assume you know what you're doing there.

Reply:
It worked without `draggable="true"` /shrug

Reply:
oh - the dragstart listener makes it draggable.